src/Generators/Microsoft.Gen.MetricsReports/Directory.Build.props

```diff
@@ -4,7 +4,6 @@
   the project itself. -->
   <PropertyGroup>
     <Stage>dev</Stage>
-    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```

Comment:
Do we also need to remove `SuppressFinalPackageVersion` here?
 https://github.com/dotnet/extensions/blob/7ebb34d4b5a0d1e18c4323cbb23a7e1b6c4622bc/src/Libraries/Microsoft.Extensions.Caching.Hybrid/Microsoft.Extensions.Caching.Hybrid.csproj#L28

Reply:
Yes, looks like I missed that. Thanks.
I found it another day performing the validations locally, but I haven't had a chance to update the PR as I'm very busy with other stuff.